lutris/gui/application.py

```diff
@@ -774,22 +774,23 @@ def on_error(game, error):
             self.quit_on_game_exit = False
         return 0
 
-    def on_settings_changed(self, dialog, state, setting_key):
+    def on_settings_changed(self, dialog, state, setting_key: str):
         if setting_key == "dark_theme":
             self.style_manager.is_config_dark = state
         elif setting_key == "show_tray_icon" and self.window:
             if self.window.get_visible():
                 self.set_tray_icon()
         return True
 
-    def on_game_start(self, game):
-        self._running_games.append(game)
+    def on_game_start(self, game_launcher: GameLauncher):
+        self._running_games.append(game_launcher.game)
         if settings.read_setting("hide_client_on_game_start") == "True":
             self.window.hide()  # Hide launcher window
         return True
 
-    def on_game_stopped(self, game):
+    def on_game_stopped(self, game_launcher: GameLauncher):
         """Callback to quit Lutris is last game stops while the window is hidden."""
+        game = game_launcher.game
```

Comment:
This, however, I like better. We want to go from the launcher to the game, not vice versa.